setup.sh

```diff
@@ -30,15 +31,20 @@ function buildWorkspace {
 	build_dir_arg=$5
 	install_dir_arg=$6
 	devel_space_only_arg=$7
-	shift 7
+	additional_options=$8
```

Comment:
to jest niepotrzebne, do wszystkich podskryptów przekazywany jest `-- "$@"` - wystarczy dodać do helpa info, że argumenty dla catkin builda trzeba poprzedzić `--` i usunąć go w wywołaniach podskryptów (a tak trzeba podać jeden dummy argument, żeby móc dać np `-w 1`)